drizzle/migrations/0005_add_documents_rag.sql

```
@@ -0,0 +1,101 @@
+-- 8. Add Storage RLS policies keyed to is_clerk_user() / owner
+DROP POLICY IF EXISTS "Allow users to upload attachments" ON storage.objects;
+CREATE POLICY "Allow users to upload attachments" ON storage.objects
+  FOR INSERT TO authenticated WITH CHECK (
+    bucket_id = 'chat-attachments'
+    AND (
+      auth.uid() = owner
+      OR public.is_clerk_user((SELECT id FROM public.users WHERE public.users.id = auth.uid()))
+    )
+  );
+
+DROP POLICY IF EXISTS "Allow users to read their own attachments" ON storage.objects;
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)